releasenotes/notes/remove-ddtrace-monkey-16e56038dc409769.yaml

```diff
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    The deprecated module ``ddtrace.monkey`` is removed.
```

Comment:
Should we include the alternative/upgrade actions to take?

e.g. module has been removed, be sure to use `ddtrace.patch/ddtrace.patch_all` instead?